2018-05-Berlin/attendee-rsvp.md

```diff
@@ -0,0 +1,90 @@
+# Node.js Collaborator Summit, 2018-05-31 - 2018-06-01, Berlin, Germany
+This document serves as an RSVP list for attendees in person, attendees remote, and the collaborator's dinner hosted by the Node Foundation.
+
+This is how we will track amount of food and seats. Submit a PR if you need to RSVP or update your RSVP.
+
+### Attendees In-Person 
+If you will be in Berlin to attend the Collaborator's Summit - add your name.
+
+1. [Myles Borins](https://github.com/MylesBorins) // Organizer
+2. [Matteo Collina](https://github.com/mcollina) // Organizer
+3. [Dan Shaw](https://github.com/dshaw) // Organizer
+4. [olivia](https://github.com/oe) // Organizer
+5. [Tracy Hinds](https://github.com/hackygolucky) // Organizer
+6. [Mark Hinkle](https://github.com/mrhinkle) // Node Foundation
+7. [Tierney Cyren](https://github.com/bnb)
+8. [Anatoli Papirovski](https://github.com/apapirovski)
+9. [Anna Henningsen](https://github.com/addaleax)
+10. [Benedikt Meurer](https://github.com/bmeurer)
+11. [Zeke Sikelianos](https://github.com/zeke)
+12. [Dhruv Jain](https://github.com/maddhruv)
```

Comment:
Awesome :heart_decoration: :heart_decoration: :heart_decoration: 
I will be there

Reply:
@maddhruv add yourself to dinner if you plan on coming pls?